communicator/models/sample.py

```diff
@@ -13,7 +13,7 @@ class Sample(db.Model):
     student_id = db.Column(db.Integer)
     computing_id = db.Column(db.String)
     date = db.Column(db.DateTime)
-    last_modified = db.Column(db.DateTime, default=datetime.utcnow, onupdate=datetime.now, nullable=False)
+    last_modified = db.Column(db.DateTime, default=datetime.utcnow, onupdate=datetime.now, nullable=True)
```

Comment:
This field must not be nulable, you may need to downgrade and then upgrade your database, but this should not be altered, I have to guarantee this value is set on all records to allow an external system to pull down information.